src/Components/WebAssembly/WebAssembly.Authentication/src/Interop/AuthenticationService.ts

```diff
@@ -233,9 +233,11 @@ class OidcAuthorizeService implements AuthorizeService {
     }
 
     private async stateExists(url: string) {
+        const stateHash = new URLSearchParams(new URL(url).hash).get('state');
         const stateParam = new URLSearchParams(new URL(url).search).get('state');
-        if (stateParam && this._userManager.settings.stateStore) {
-            return await this._userManager.settings.stateStore.get(stateParam);
+        const state = stateHash ?? stateParam;
```

Comment:
Can we insert this order? It's less breaking for users upgrading if the `state` param` appears in both places.
```suggestion
        const state = stateParam ?? stateHash;
```